### DISCUSSION TITLE
Disable Typing of Signature Completions

### DISCUSSION THREAD

**Discussion by ibrokemypie** [microsoft/pylance-release#4595] [Enhancement]
**Is your feature request related to a problem? Please describe.**
When I autocomplete a signature such as `def __init__` the snippet inserted includes type hinting such as `-> None`. In most of my codebases I work on I do not have typing set up, so I have to then delete this type hinting every time the snippet is inserted which is a pain. I can't find any option to disable the type hinting.

**Describe the solution you'd like**
An option to disable the type hinting in signature snippet completions.

**Additional context**
![image](https://user-images.githubusercontent.com/7827846/235383332-d0beeb7a-dcfc-4ba0-bd6a-acf14d3ab930.png)
![image](https://user-images.githubusercontent.com/7827846/235383338-24f537a1-a429-46ea-8c5c-7f04f1fe0b3b.png)

Seems to have been brought up before at https://github.com/microsoft/pyright/discussions/3383

**Comment by erictraut** [comment]
Your feature request is related to a language server feature, not a core type checking feature, so it's under the purview of the pylance team. @rchiodo, could you please transfer this to the pylance-release project and handle it as you see fit?

**Comment by erictraut** [comment]
@ibrokemypie, when you say that you "don't have typing set up", what do you mean? Type annotations don't require any special "setup" in Python. They can be added to any Python file. When present, they serve as documentation that is useful to both human readers and tools like type checkers and language servers. Are you saying that you don't want static type annotations in your code despite these benefits?

**Comment by ibrokemypie** [comment]
I mean that codebases at work are not using typing as they are quite old, so spotty type hinting only in files that I touch wouldn't make it through review due to inconsistency. Not arguing that its useful, and I wish this wasn't the case, but unfortunately it is. Don't see the issue with an option to disable if the user sees the need
